lib/utils/fs_test.go

```diff
@@ -90,3 +92,54 @@ func TestLocks(t *testing.T) {
 
 	require.NoError(t, unlock())
 }
+
+func TestOverwriteFile(t *testing.T) {
+	have := []byte("Sensitive Information")
+
+	tmpFile, err := os.CreateTemp("", "teleport-overwrite-file-test-")
```

Comment:
Nit:  I'd be tempted to use `t.TempDir()` here and then create a file under that path.

Reply:
Good suggestion. I've made the change.